### DISCUSSION TITLE
Package installation causing compile error

### DISCUSSION THREAD

**Discussion by lmcglynn** [ThibaultJanBeyer/DragSelect#230] [Q&A]
Hello,
I'm currently working on a GUI that requires drag-selection functionality and decided to try to use this package due to its detailed documentation. However, when I installed it into my React JS app (using Node v14) using `yarn add dragselect` and then `yarn start`, this package caused the compile error as seen in the attached screenshot.
<img width="911" alt="AGV_vUeo-EFD5LBhjduOc3PEFBRkLSg1xOmf11EZ-VOskSJxe_-Av6jpYp3pzV8iukNy3ufDW1VsRT7euvqJLlLwHaELtBYh9pzquSTU5NvEovzpQsjbqoTy-iSN" src="https://github.com/user-attachments/assets/04384c2a-263c-4781-a8b0-a06de7e0f060">
I have already found this [discussion](https://stackoverflow.com/questions/52237855/support-for-the-experimental-syntax-classproperties-isnt-currently-enabled) and have tried nearly all of the suggested fixes but the error has not changed or gone away. I was wondering if anyone else has run into this issue with this package in particular and if they were able to work around it. Any help is much appreciated.

**Comment by ThibaultJanBeyer** [comment]
Thank @lmcglynn for taking the time to write this issue and try the library.
Uh oh that sounds weird, as we're using DragSelect internally in a react project that is the www.dragselect.com website and there is no compile issues. 

I also tried quickly creating a new [codesandbox](https://codesandbox.io/p/sandbox/qr6llq?file=%2Fpackage.json) with react and adding DragSelect, which works fine.

So I guess this is not happening anymore? What would be most helpful would be a codesandbox that reproduces the issue. Thank you ❤️
